<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logInForm</class>
 <widget class="QWidget" name="logInForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LoginEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 20pt &quot;黑体&quot;;
</string>
   </property>
   <property name="text">
    <string>系统账号登录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>112</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePassWord">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loginbtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLog">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt ;</string>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>365</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空所有</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signBtn">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>365</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>98</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;
</string>
   </property>
   <property name="text">
    <string>账户</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwdlabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>165</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wrongEdit2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>225</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;
color: rgb(255, 60, 43);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="wrongEdit1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>156</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;
color: rgb(255, 60, 43);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="loginwidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="backwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>391</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="minisizeBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backwidget</zorder>
  <zorder>LoginEdit</zorder>
  <zorder>lineName</zorder>
  <zorder>linePassWord</zorder>
  <zorder>loginbtn</zorder>
  <zorder>labelLog</zorder>
  <zorder>forgetBtn</zorder>
  <zorder>signBtn</zorder>
  <zorder>namelabel</zorder>
  <zorder>pwdlabel</zorder>
  <zorder>wrongEdit2</zorder>
  <zorder>wrongEdit1</zorder>
  <zorder>loginwidget</zorder>
  <zorder>minisizeBtn</zorder>
  <zorder>closeBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
